ten-simple-rules-dockerfiles.Rmd

```diff
@@ -505,12 +509,12 @@ This modularization is a potential risk for understandability and consistency, w
 You can also use package managers to install software from source code `COPY`ied into the image (see&nbsp;\ruleref{rule:mount}).
 And finally, you can use many package managers to install software from source obtained from code management repositories, e.g., installing a specific tool identified by a GitHub version tag or commit hash.
 Be aware of the risk of depending on such repositories, especially if they are out of your control, although the installation command with a full URL should allow readers of your `Dockerfile` to dig deeper if problems arise.
-Such an installation from source gives you a lot of freedom, including the one to shoot yourself in the foot.
+Such an installation from source gives you a lot of freedom, including to shoot yourself in the foot!
```

Comment:
I think the reference "to the one" is needed here - what do others think?

Reply:
I think if you'd like to have "the one" it should be "gives you many freedoms, including the one to shoot yourself in the foot!" since you are referring to one of many distinct freedoms.

Reply:
Yes this works!

Reply:
Done!